### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Gem-AaqilALHassan**

### CODE REVIEW

File: Exception_demo.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+print("Enter the num1")
+num1 = int(input())
+print("Enter the num2")
+num2 = int(input())
+try:
+    print("the sum of these two numbers is", num1+num2)
+except xception as e:
```
Comment (kanay-gemini) [inline Exception_demo.py]: This will throw an error as exception spelling is wrong, fix it please

File: Exception_demo.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#Python 3
+
+a = [1, 2, 3]
+try:
+	print ("Second element = %d" %(a[1]))
+
+	# Throws error since there are only 3 elements in array
+	print ("Fourth element = %d" %(a[3]))
+
+except:
```
Comment (kanay-gemini) [inline Exception_demo.py]: this is wrong practice of writing except clause without defining a exception it is going to handle

File: Exception_demo.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+try:
+	fun(3)
+	fun(5)
+
+# note that braces () are necessary here for
+# multiple exceptions
+except ZeroDivisionError:
+	print("ZeroDivisionError Occurred and Handled")
+except NameError:
+	print("NameError Occurred and Handled")
```
Comment (kanay-gemini) [inline Exception_demo.py:47]: also add the except Exception as e: at the end to avoid any unxpected error to happen

**Comment by Gem-AaqilALHassan** [comment]
Good morning Kanay ,

All changes are updated, sorry for inconvenience caused.

Regards
Aaqil AL Hassan
________________________________

**Comment by kanay-gemini** [comment]
No problem, I have merged the MR, Continue your learning and let me know in case of any concerns.

Thanks,
Kanay Bhandari
________________________________

**Pull request merged**